profiles/models.py

```diff
@@ -77,8 +77,12 @@ class Profile(models.Model):
         default=None,
         null=True,
     )
-    edrpou = models.IntegerField(
-        unique=True, validators=[validate_edrpou], default=None, null=True
+    edrpou = models.CharField(
+        max_length=8,
+        unique=True,
+        validators=[validate_edrpou],
+        default=None,
+        null=True,
```

Comment:
It's usually not a great idea to make nullable field unique. As only one record with null will fit.

Reply:
We need to have this field both unique and nullable. As it's not obligatory field during registration and user could fill it later (we have multiple null edrpou fields in db now).

Reply:
blank=True was added as recommended in documentation for this case

Reply:
@Lvyshnevska wow, if it actually works this way — this is cool. Probably I missed it among PostgreSQL updates, there was a time when it didn't work (you could put only one record with `null` if the field had 'unique' constraint).